duo_client/admin.py

```diff
@@ -519,6 +528,106 @@ def get_authentication_log(self, api_version=1, **kwargs):
                 row['host'] = self.host
         return response
 
+    def get_activity_logs(self, **kwargs):
+        """
+        Returns activity log events.
+
+        As of now, the activity endpoint is not in general availability and is restricted to a few customers for private preview.
+        If you have any questions or need more information, feel free to reach out to support for guidance.
+
+
+        mintime (required) - Unix timestamp in ms; fetch records >= mintime
```

Comment:
If we’re setting a default we should remove the `(required)` comment from `mintime` and `maxtime`